client/src/TenderList.js

```diff
@@ -19,10 +20,19 @@ const TendersList = () => {
 	const fetchTenders = async (page) => {
 		setLoading(true);
 		try {
-			const data = await get(`/api/tenders?page=${page}`);
-			if (data && data.results && data.pagination) {
-				setTenders(data.results);
-				setPagination(data.pagination);
+			const tenderData = await get(`/api/tenders?page=${page}`);
+			const bidsData = await get("/api/bidder-bid?page=1");
+
+			if (
+				tenderData &&
+				tenderData.results &&
+				tenderData.pagination &&
+				bidsData &&
+				bidsData.results
+			) {
```

Comment:
```suggestion
			if (tenderData) {
```

`results` and `pagination` are always included for a `200` response. We don't need to explicitly check for them